### DISCUSSION TITLE
Observations in terms of WMO WW weather interpretation codes

### DISCUSSION THREAD

**Discussion by Nomlax** [weather-gov/api#572] [Q&A]
Hi,

I'm a bit new with using the NWS weather API, but I need weather data from multiple countries in a standardized format for a project.

I figured that the `ww` codes ([WMO](https://library.wmo.int/doc_num.php?explnum_id=10235) 4677) would be a good way to standardize weather observations since a couple of other countries' weather services use that standard. As far as I can see, there is no sign of `ww` codes being used in NWS observations. (I'm using the `stations/{station}/observations/latest` endpoint)

All I've found about `ww` codes referenced anywhere in the NWS and NOAA website is one reference page [here](https://www.nodc.noaa.gov/archive/arc0021/0002199/1.1/data/0-data/HTML/WMO-CODE/WMO4677.HTM)

Does the NWS use `ww` codes? Is there a alternative (or any) standard that the NWS uses? Is there a way to "convert" NWS observations to `ww` codes?

Any help would be appreciated. Thank you.

**Comment by dwhitemv25** [comment]
As we say in the information technology world: The great thing about standards is that there's so many to choose from... 

Could you tell us more about what weather data you're seeking?

If airport weather reports (METAR) work for you, those are available from the API, even for non-US locations. I tested CYYZ (Toronto) and VHHH (Hong Kong) and both are in there. Looks like the API even parses it for you. If you need some custom handling, though, writing a METAR parser takes some experience, but it's been done many times in open-source code.

Otherwise, a weather services company can aggregate data sources and perform normalization of the observations.
